### ISSUE TITLE
Error spam in Sublime console

### ISSUE THREAD

**Issue by buildersbrewery** [SublimeText/TrailingSpaces#123]
TrailingSpaces keeps spamming my Sublime console with the following error:

```text
Traceback (most recent call last):
  File "..\sublime3\sublime_plugin.py", line 389, in run_callback
    expr()
  File "..\sublime3\sublime_plugin.py", line 496, in <lambda>
    run_callback('on_modified', callback, lambda: callback.on_modified(v))
  File "trailing_spaces in ..\sublime3\Data\Installed Packages\TrailingSpaces.sublime-package", line 408, in on_modified
  File "trailing_spaces in ..\sublime3\Data\Installed Packages\TrailingSpaces.sublime-package", line 129, in match_trailing_spaces
  File "trailing_spaces in ..\sublime3\Data\Installed Packages\TrailingSpaces.sublime-package", line 86, in find_trailing_spaces
  File "..\sublime3\sublime.py", line 649, in __getitem__
    raise IndexError()
IndexError
```
---

* Sublime Text Build 3156 portable x64
* Win10 Pro x64 1709

**Comment by davidfromtheeast** [comment]
+1

**Comment by karolyi** [comment]
+1

**Comment by karolyi** [comment]
+1

**Comment by Kamik423** [comment]
+1

**Comment by michaelblyons** [comment]
If you update the package and reload ST, does this still happen?

**Comment by rchl** [comment]
I think this is crash on getting `view.sel()[0]` and should be resolved by #131
